src/main/java/org/example/Homework11.java

```diff
@@ -0,0 +1,45 @@
+package org.example;
+
+import java.util.*;
+
+public class Homework11 {
+
+    public Integer maxDuplicate(List<Integer> nums) {
+        List<Integer> uniqueNumbers = new ArrayList<>();
+        List<Integer> duplicates = new ArrayList<>();
+        for (Integer num : nums) {
+            if (uniqueNumbers.contains(num)) {
+                duplicates.add(num);
+            } else {
+                uniqueNumbers.add(num);
+            }
+        }
+        if (duplicates.isEmpty()) {
+            return null;
+        }
+        return (Integer) Collections.max(duplicates);
+    }
+
+    public List<String> wordNumeration(List<String> words) {
+        List<String> numberedWords = new ArrayList<>();
+        HashMap wordCounts = new HashMap<>();
```

Comment:
Not `HashMap` but `Map<String, Integer>`. Raw usage is discouraged, look at the Intellij's warnings as well. Also, it's a better practice to use Interface as a type